### PR TITLE
Fixes mesh controls to orbit COM of meshes

### LINKED ISSUE #118: Controls after adding mesh still orbit around origin
Annoying that I have to search for the mesh I just put in

### PR DESCRIPTION
**Pull request by Raphtor**
Fixes #118

### CODE REVIEW

File: pytri/__init__.py [hunk 1]
```diff
@@ -515,4 +520,6 @@ def mesh(self, *args, **kwargs):
         color = kwargs.get("color", "#00bbee")
 
         mesh = Mesh(geometry=geo, material=MeshLambertMaterial(color=color))
-        return self._add_layer(mesh)
+        ret = self._add_layer(mesh)
+        self._reposition_camera_on_bbs()
```
Comment (j6k4m8) [inline pytri/__init__.py:524]: I don't love the idea of repositioning the camera without the user asking for it. I was thinking:

```python
from pytri import Figure

fig = Figure(auto_camera=True)
fig.blah()
```

or,

```python
from pytri import Figure

fig = Figure()
fig.blah()
fig.recenter_camera()
```
?

**Comment by Raphtor** [comment]
Fixes #114, but doesn't solve scaling.

File: pytri/__init__.py [hunk 2]
```diff
@@ -518,8 +530,11 @@ def mesh(self, *args, **kwargs):
...
         color = kwargs.get("color", "#00bbee")
-
-        mesh = Mesh(geometry=geo, material=MeshLambertMaterial(color=color))
+        alpha = kwargs.get('alpha', 1.)
+        transparent = alpha != 1.
+        mat = MeshLambertMaterial(color=color, alpha=alpha, transparent=transparent)
+        mesh = Mesh(geometry=geo, material=mat)
         ret = self._add_layer(mesh)
-        self._reposition_camera_on_bbs()
+        if kwargs.get("auto_camera", False):
```
Comment (j6k4m8) [inline pytri/__init__.py:538]: Not to be a pain, but:

* If a user provides `auto_camera` to a function, it makes sense for the operation to be scoped to objects added in that function call
* If a user provides `auto_camera` to the _figure_ (this is what i was saying in the prev comment) it maybe makes more sense for the operation to be scoped to the full scene (as it is here as-is, I think?)

**Comment by Raphtor** [comment]
Closed in favor of #120

**Pull request closed without merging**